src/signals/IncidentMap/components/AddressLocation/AddressSearchMobile.tsx

```diff
@@ -51,6 +51,8 @@ export const AddressSearchMobile = ({
             variant="blank"
           />
           <StyledPDOKAutoSuggest
+            aria-label="searchAddressBarMob"
```

Comment:
these don't need to be camelcase an can be descriptive with spaces

Reply:
Done